redhat-release-coreos.spec

```diff
@@ -94,6 +94,10 @@ for file in RPM-GPG-KEY* ; do
     gpg --dearmor < $file > %{buildroot}/%{_datadir}/ostree/trusted.gpg.d/$file.gpg
 done
 
+# setup default PATH
+# https://github.com/openshift/os/issues/191
+echo 'PATH=/usr/bin:/bin:/usr/sbin:/sbin' > %{buildroot}/etc/profile.d/path.sh
```

Comment:
This is missing the `/usr/local` variants too, right? E.g. something like:

```
/usr/local/sbin:/usr/local/bin:...
```

?

I guess a case could be made for dropping them on RHCOS given that we want to discourage SSH'ing, though it can still be useful for debugging purposes.

Reply:
Yeah, I was going off of the diff with and without this patch in https://github.com/openshift/os/issues/191#issuecomment-409297511 and it seems like those just get dropped.

Actually, there's a `pathmunge` function provided by `/etc/profile` that could make this easier, e.g.:

```
cat > %{buildroot}/etc/profile.d/path.sh <<EOF
pathmunge /usr/local/sbin
pathmunge /usr/sbin
EOF
```

That should have no effect on root logins which already have those in their paths, but still affect nonroot logins.

Reply:
Right, I saw that function.  I'll tinker with that approach and see how it looks; seems promising!